src/locales/en.json

```diff
@@ -1153,6 +1208,7 @@
     "type": {
       "AWS": "AWS",
       "AZURE": "Microsoft Azure",
+      "GCP": "GCP",
```

Comment:
This shouldn't be abbreviated. The AWS probably shouldn't be either?

Except for source_details.type and source_details.filter, looks like a lot of dead code again.

Reply:
Done, we will change AWS abbreviation in another PR as some of it will be removed as dead code